sovrin

```diff
@@ -1,78 +0,0 @@
-#! /usr/bin/env python3
```

Comment:
Well, this script was placed under sovrin-client/sovrin_client folder, while the script which we use is placed under sovrin-client/scripts (though content were same for both), so I deleted this one.